### PR TITLE
Changed safety approach to distinguish between install and development

### PR DESCRIPTION
**Pull request by anilak11**
Details:

* Split the safety runs between requirements file for installation and minimum-constraints file for development. The safety run for installation must succeed, while the safety run for development shows its issues, but is allowed to fail.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -465,21 +466,36 @@ $(done_dir)/install_pip_$(pymn)_$(PACKAGE_LEVEL).done: Makefile
...
 	-$(call RM_FUNC,$@)
-	safety check --policy-file $(safety_policy_file) -r minimum-constraints.txt --full-report
+	-safety check --policy-file $(safety_all_policy_file) -r minimum-constraints.txt --full-report
 	echo "done" >$@
 	@echo "Makefile: Done running Safety"
 endif
 endif
 
+$(done_dir)/safety_install_$(pymn)_$(PACKAGE_LEVEL).done: $(done_dir)/develop_$(pymn)_$(PACKAGE_LEVEL).done Makefile $(safety_install_policy_file) requirements.txt
+ifeq ($(python_m_version),2)
```
Comment (andy-maier) [inline Makefile:485]: This should be `python_major_version` instead.

File: Makefile [hunk 2]
```diff
@@ -465,21 +466,36 @@ $(done_dir)/install_pip_$(pymn)_$(PACKAGE_LEVEL).done: Makefile
...
+	@echo "Makefile: Warning: Skipping Safety for install packages on Python $(python_version)" >&2
+else
+ifeq ($(python_mn_version),3.5)
```
Comment (andy-maier) [inline Makefile:488]: This should be `python_m_n_version` instead.

**Review by andy-maier — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (anilak11) [reply Makefile]: copy paste error,
I changed it now

File: Makefile (hunk 2 above)
Comment (anilak11) [reply Makefile]: Done

**Review by andy-maier — APPROVED** [review]

**Pull request merged**